### DISCUSSION TITLE
关于文件的输入

### DISCUSSION THREAD

**Discussion by odd-clover** [liusy58/NJU_AdvancedProgramming_SP22#93] [Q&A] [answered]
使用FILE类型的fwrite，fputs等函数对文件是覆盖式写法，改变了一个地方的长度后后续内容都需要重新覆盖。
有没有办法能够修改一个地方的长度后，让后续内容自动前移或后移？

**Comment by RicoloveFeng** [comment, marked as answer]
貌似没有，只能把所有内容读进内存，然后插入后写回
